<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainWindow</class>
 <widget class="QWidget" name="ClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FChat - Client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="server_ip_label">
       <property name="text">
        <string>Server IP : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="server_ip_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="server_port_label">
       <property name="text">
        <string>Server Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="server_port_spinBox">
       <property name="maximum">
        <number>60000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>85</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connection_pushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="sent_message_textBrowser"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pseudo_label">
       <property name="text">
        <string>Pseudo : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pseudo_lineEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="message_label">
       <property name="text">
        <string>Message : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="message_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="send_pushButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
